BibBuddy/app/src/main/java/de/bibbuddy/IsbnRetriever.java

```diff
@@ -0,0 +1,119 @@
+package de.bibbuddy;
+
+import java.io.StringReader;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.xml.sax.InputSource;
+
+public class IsbnRetriever implements Runnable {
+  private final String apiUrl = "https://lod.b3kat.de/";
+  private final String apiXmlParameter = "?output=xml";
+  private final String isbnApi = apiUrl + "data/isbn/%s" + apiXmlParameter;
+  private final String isbn;
+  private Book book = null;
+
+  /**
+   * The IsbnRetriever class connects to the API and returns metadata for an ISBN.
+   *
+   * @param isbn ISBN from which metadata should be returned
+   * @author Luis Moßburger
+   */
+  IsbnRetriever(String isbn) {
+    this.isbn = isbn;
+  }
+
+  private static Document loadXmlFromString(String xml) throws Exception {
+    DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+    DocumentBuilder builder = factory.newDocumentBuilder();
+    InputSource is = new InputSource(new StringReader(xml));
+    return builder.parse(is);
+  }
+
+  private Book createRecord(Document xmlMetadata) {
+    long id = 0L;
+    String isbn = xmlMetadata.getElementsByTagName("bibo:isbn").item(0).getTextContent();
+    String title = xmlMetadata.getElementsByTagName("dc:title").item(0).getTextContent();
+    String subttle = xmlMetadata.getElementsByTagName("dc:title").item(0).getTextContent();
```

Comment:
Why is the subtitle the same as the title?
```suggestion
    String subtitle = xmlMetadata.getElementsByTagName("dc:title").item(0).getTextContent();
```

Reply:
Mistake from me, thanks for pointing that out!